includes/class-newspack-popups-segmentation.php

```diff
@@ -309,7 +309,43 @@ public static function create_database_table() {
 	 * Get all configured segments.
 	 */
 	public static function get_segments() {
-		return get_option( self::SEGMENTS_OPTION_NAME, [] );
+		$segments                  = get_option( self::SEGMENTS_OPTION_NAME, [] );
+		$segments_without_priority = array_filter(
+			$segments,
+			function( $segment ) {
+				return empty( $segment['priority'] );
```

Comment:
This filter will always return at least one element, since one segment will have priority of `0` (which will satisfy the `empty()` condition).

Reply:
Ooh, good catch. I think I actually want `! isset` instead, right? Updated in2958a5e.